src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.spec.ts

```diff
@@ -1050,9 +1045,13 @@ describe('PoolFormComponent', () => {
         expect(form.getValue('ratio')).toBe(pool.options.compression_required_ratio);
       });
 
-      it('is only be possible to use the same or more pgs like before', () => {
+      it('should pg update should work on the first change not the second', () => {
```

Comment:
Is the first 'should' on purpose?

```suggestion
      it('pg update should work on the first change not the second', () => {
```